en/docs/learn/data-publishing-reports.md

```diff
@@ -113,10 +113,12 @@ following reports according to the OBIE templates:
 | 6 | Long-lived consents - Outstanding consents BoP | LONG_LIVED_CONSENT_ADOPTION | |
 | 7 | Long-lived consents - Revoked consents | LONG_LIVED_CONSENT_ADOPTION | |
 | 8 | Long-lived consents - Expired consents | LONG_LIVED_CONSENT_ADOPTION | |
-| 9 | Long-lived consents - Renewed consents | LONG_LIVED_CONSENT_ADOPTION | |
-| 10 | Long-lived consents - New consents | LONG_LIVED_CONSENT_ADOPTION | |
-| 11 | Long-lived consents - Outstanding consents EoP | LONG_LIVED_CONSENT_ADOPTION | |
-| 12 | Version | LONG_LIVED_CONSENT_ADOPTION | |
+| 9 | Long-lived consents - Consents Requiring Reauthentication | CONSENT_REQ_REAUTHENTICATION | This is the total number of unique Long-lived consents which required 90 days PSU re-authentication. Join ACCOUNTS_RAW_DATA and API_INVOCATION_RAW_DATA tables. And get the CONSENT_ID count AS TOTAL_CONSENT_REQ_REAUTH where (CONSENT_TYPE = 'long-lived' and AUTHORISATION_STATUS = 'Authorised' and RE_AUTHORISATION == false) having (EXPIRATION_DATE_TIME > REPORTING_DATE_TIME and REPORTING_DATE_TIME - 90 days> AUTHORIZED_TIME_DATE) |
+| 10 | Long-lived consents - Renewed consents | LONG_LIVED_CONSENT_ADOPTION | |
+| 11 | Long-lived consents - New consents | LONG_LIVED_CONSENT_ADOPTION | |
+| 12 | Long-lived consents - Outstanding consents EoP | LONG_LIVED_CONSENT_ADOPTION | |
+| 13 | Long-lived consents - Total Refreshed consents | REFRSHED_CONSENT | This is the total number of unique Long-lived consents which have been refreshed (Re-authenticated). Join ACCOUNTS_RAW_DATA and API_INVOCATION_RAW_DATA tables. Get the CONSENT_ID count AS TOTAL_CONSENT_REQ_REAUTH where (CONSENT_TYPE = 'long-lived' and AUTHORISATION_STATUS = 'Authorised' and RE_AUTHORISATION == true) having (EXPIRATION_DATE_TIME > REPORTING_DATE_TIME and REAUTHORIZED_DATE_TIME > AUTHORIZED_TIME_DATE + 90 days) |
```

Comment:
'Join ACCOUNTS_RAW_DATA and API_INVOCATION_RAW_DATA tables. Get the CONSENT_ID count AS TOTAL_CONSENT_REQ_REAUTH where (CONSENT_TYPE = 'long-lived' and AUTHORISATION_STATUS = 'Authorised' and RE_AUTHORISATION == true) having (EXPIRATION_DATE_TIME > REPORTING_DATE_TIME and REAUTHORIZED_DATE_TIME > AUTHORIZED_TIME_DATE + 90 days) |'

Same as above. Feels like an SQL query.

Reply:
Fixed in 34b67885e2c5e89115e1466d8b58ad4f07742a8e.